aries-site/src/examples/templates/forms/RequiredFieldsExample.js

```diff
@@ -80,48 +63,43 @@ export const RequiredFieldsExample = () => {
           value={formValues}
           onChange={onFormChange}
         >
-          <RequiredFormField
+          <FormField
             required
             error="Provide a unique name."
             htmlFor="name__input"
             name="name"
             label="Name"
           >
             <TextInput id="name" name="name" />
-          </RequiredFormField>
-          <RequiredFormField
+          </FormField>
+          <FormField
             required
             htmlFor="superPower__input"
             name="superPower"
             label="Superpower"
           >
             <Select options={superPower} id="superPower" name="superPower" />
-          </RequiredFormField>
-          <RequiredFormField
+          </FormField>
+          <FormField
             required
             htmlFor="weakness__input"
             name="weakness"
             label="Weakness"
           >
             <Select options={weakness} id="weakness" name="weakness" />
-          </RequiredFormField>
-          <RequiredFormField
-            required
-            htmlFor="email__input"
-            name="email"
-            label="Email"
-          >
+          </FormField>
+          <FormField required htmlFor="email__input" name="email" label="Email">
             <TextInput id="email" name="email" />
-          </RequiredFormField>
-          <RequiredFormField
+          </FormField>
+          <FormField
             required
             help="Would you like to apply nemesis character"
             htmlFor="nemesis__input"
             name="nemesis"
             label="Nemesis"
           >
```

Comment:
Just noticing this, but in this instance where we are asking "Would you like to apply nemesis character?", it feels awkward to have the field required, especially since the input is a toggle. I think we can remove the required without hurting the example.
```suggestion
          <FormField
            help="Would you like to apply nemesis character?"
            htmlFor="nemesis__input"
            name="nemesis"
            label="Nemesis"
          >
```